src/sonic-config-engine/minigraph.py

```diff
@@ -1566,6 +1566,10 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
     if is_storage_device:
         results['DEVICE_METADATA']['localhost']['storage_device'] = "true"
 
+    # clean vlan members field before final assignment
+    for vlan_name in vlans.keys():
```

Comment:
Just curious, why do we need to populate this in `parse_dpg()` to be removed later? Is it used somewhere else?

Reply:
could not find a place where it is used. I can remove it all together, after keeping a comment about use of "Attach To", if in future somebody tries to reuse it for any purpose. Thx.